backup-and-restore.adoc

```diff
@@ -0,0 +1,130 @@
+= Backup and restore Taiga
+:toc: left
+:toclevels: 1
+:numbered:
+:source-highlighter: pygments
+:pygments-style: friendly
+
+== Recommended backup process for Taiga6 docker
```

Comment:
to enable anchors, you can see the example in `setup-production`.

```
                                   
[[setup-prod-with-docker]]         
== Docker                          
```

Reply:
Done ✅